Modules/CommandHandler.cs

```diff
@@ -66,6 +67,7 @@ private async Task HandleCommand(SocketMessage s, DiscordSocketClient client)
 
             // Check if there is a custom command registered under the name
             string commandName = msg.Content.Substring(argPos).Split(' ')[0].ToLowerInvariant();
+            string convertedCommand = CommandConversion.ToConverted(msg.Content.Substring(argPos));
```

Comment:
msg.Content.Substring is used multiple times with the same parameters, make a variable for it.